README.md

```diff
@@ -2,9 +2,9 @@
 [![Build Status](https://travis-ci.org/ustwo/baseview-swift.svg?branch=master)](https://travis-ci.org/ustwo/baseview-swift)
 [![codecov.io](https://codecov.io/github/ustwo/baseview-swift/coverage.svg?branch=master)](https://codecov.io/github/ustwo/baseview-swift?branch=master)
 
-# BaseView
+# BaseViewSwift
 
-The BaseView framework provides an organizational tool for writing custom views using `UIKit`. This framework is written using Swift 2.1. It has been developed and used by iOS developers at [ustwo](ustwo.com).
+The BaseViewSwift framework provides an organizational tool for writing custom views using `UIKit`. This framework is written using Swift 2.1. It has been developed and used by iOS developers at [ustwo](ustwo.com).
```

Comment:
Hmm I'm not sure how I feel about the framework itself being called BaseViewSwift :D We have an Obj-C version of this lib and the -swift in the repo name was just to help identify against that. Whats the motivation behind suffixing? Is it for identification purposes (in which case US2 might help us?)